packages/wallet/scripts/start.js

```diff
@@ -83,7 +84,27 @@ void (async () => {
   }
 
   console.log(chalk.cyan("Starting ganache and deploying contract..."));
-  ganacheServer = await startGanacheAndDeploy();
+  ganacheServer = new GanacheServer(
+    Number(process.env.GANACHE_PORT),
+    Number(process.env.CHAIN_NETWORK_ID)
+  );
+  await ganacheServer.ready();
+
+  const {
```

Comment:
I wonder if it's worth it to consolidate this logic and the logic in contract-test-setup.